<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Custom_Trip</class>
 <widget class="QDialog" name="Custom_Trip">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>303</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>303</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(197, 15, 35);</string>
  </property>
  <layout class="QFormLayout" name="formLayout_2">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0)</string>
     </property>
     <property name="title">
      <string>Custom Start</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QComboBox" name="cStart_Combo">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(239, 172, 1)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="cStart_Button">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(239, 172, 1);</string>
          </property>
          <property name="text">
           <string>Start Here!</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Custom Trip</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QComboBox" name="cTrip_Box">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(239, 172, 1);</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="Add_cTrip_Button">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(239, 172, 1);</string>
          </property>
          <property name="text">
           <string>Add To Trip</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <widget class="QTableView" name="cTrip_Table">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="cTrip_Start_Button">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(239, 172, 1);</string>
        </property>
        <property name="text">
         <string>Start Trip!</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="Back_Button">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(239, 172, 1);</string>
     </property>
     <property name="text">
      <string>Go Back</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
